src/js/classes/AjaxBootstrapSelectList.js

```diff
@@ -55,15 +56,27 @@ var AjaxBootstrapSelectList = function (plugin) {
             text: $option.text(),
             'class': $option.attr('class') || '',
             data: $option.data() || {},
-            preserved: plugin.options.preserveSelected,
```

Comment:
@RomanRabirokh why did you change this line? I think the rest of this makes sense for the most part, but not sure why you did this.

Reply:
Oh. it was a long time ago. now hard to remember.

I have now tested, it does not affect anything.

But it is better to use your option. Because I do not exclude that this is my mistake.

Reply:
I agree, this shouldn't be changed.